ci/upgrade/cstor/sanity.sh

```diff
@@ -15,13 +15,10 @@
 # limitations under the License.
 
 
-set -ex
+set -x
```

Comment:
Why exit on failure was removed?

Reply:
If the test fails at line 64 the script exists the job logs don't get printed to debug the issue